spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-binder-functional-sample/spring-cloud-gcp-pubsub-stream-binder-functional-sample-sink/src/main/resources/application.properties

```diff
@@ -5,7 +5,8 @@
 spring.cloud.stream.function.bindings.logUserMessage-in-0=input
 spring.cloud.stream.bindings.input.destination=my-topic
 
-spring.cloud.function.definition=logUserMessage
+# Optional, as Spring Cloud Stream will autodiscover the correct functional bean.
+#spring.cloud.function.definition=logUserMessage
```

Comment:
maybe we should just add comments saying this property is optional, but keep it configured?

Reply:
Yeah, I think I'll do that.